.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: node_js
 node_js:
-  - "10"
```

Comment:
It shouldn't be removed. Instead use `this.skip()` to skip particular test-case.
See more: https://mochajs.org/#inclusive-tests

Reply:
Hi @climba03003 thanks for your feedback. As I don't know the root cause of the tests failing in v12, I am just re-adding v10 as suggested.